static/templates/topic_sidebar_actions.hbs

```diff
@@ -49,6 +49,16 @@
             {{t "Move topic"}}
         </a>
     </li>
+    <li>
+        <a tabindex="0" class="sidebar-popover-toggle-resolved" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
+            <i aria-hidden="true">✔️ </i>
```

Comment:
This character is wider than the FontAwesome icons and breaks the alignment of menu entries.

Reply:
They look perfectly aligned to me
<img width="595" alt="Screenshot 2021-06-15 at 2 25 45 PM" src="https://user-images.githubusercontent.com/25124304/122023990-abe44200-cde5-11eb-8612-544044b18225.png">

Reply:
<img width="595" alt="Screenshot 2021-06-15 at 2 27 50 PM" src="https://user-images.githubusercontent.com/25124304/122024261-e352ee80-cde5-11eb-82b6-24d186b3ddef.png">

Ah! It does look misaligned on Firefox.

Reply:
Color is not perfect is both the images too. Working on a fix....

Reply:
<img width="900" alt="Screenshot 2021-06-15 at 2 33 56 PM" src="https://user-images.githubusercontent.com/25124304/122025137-bb17bf80-cde6-11eb-9cf9-fb0c09e22be8.png">

I changed to code to use `fa-check`.

Reply:
<img width="900" alt="Screenshot 2021-06-15 at 2 34 31 PM" src="https://user-images.githubusercontent.com/25124304/122025267-db477e80-cde6-11eb-9f57-7f92ecb58c55.png">
<img width="900" alt="Screenshot 2021-06-15 at 2 34 42 PM" src="https://user-images.githubusercontent.com/25124304/122025276-de426f00-cde6-11eb-8a9d-68f2a67aeb82.png">

Looks perfect on both the browsers now.